.github/workflows/go.yml

```diff
@@ -10,8 +10,25 @@ permissions:
   contents: read
 
 jobs:
+  conditional-changes:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    outputs:
+      doc: ${{ steps.filter.outputs.doc }}
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        filters: |
+          doc:
+            - 'site/**'
+
   lint:
     name: Lint
+    needs: conditional-changes
+    if: needs.conditional-changes.outputs.doc == 'false'

```

Comment:
@sarab97 
Thanks for your contribution!

So, I think it looks like this condition checks only whether `site/` was changed or not.
I worry that maybe this condition won't work with any PR changing code and documents.
Please tell me if I misunderstood your work!

Reply:
As per my understanding of README from https://github.com/dorny/paths-filter. The condition will check for change and as per filter condition ensure the change is only in `site` directory. So if both document and code is changed it should evaluate to `false`.

Reply:
https://github.com/dorny/paths-filter?tab=readme-ov-file#outputs
> 'false' - if none of changed files matches any of filter rules

Oh. Thanks I understand.
I misunderstand `false` as meaning the reverse of `true`.